README.md

```diff
@@ -109,7 +118,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+filename: A2.3.pdf
```

Comment:
Very good. Only one problem: you have unnecessary state, as you can merge S1 and S2:

![image](https://user-images.githubusercontent.com/10648/78732283-d98f4a80-7907-11ea-8b45-c7cea42315f1.png)

# -2